src/errors.ts

```diff
@@ -0,0 +1,48 @@
+import { FastifyReply } from 'fastify';
+
+export function handleServerError(reply: FastifyReply, message: string, code: number = 500) {
+  const statusCode = code || reply.statusCode;
+  const response = message || 'The requested route was  not found';
+  return reply.status(statusCode).send({
+    sucess: false,
+    message: response,
+  });
+}
+
+//  A data about probable errors might need it later
+
+export const STANDARD = {
+  CREATED: 201,
+  SUCCESS: 200,
+  NOCONTENT: 204,
+};
+
+export const ERROR404 = {
+  statusCode: 404,
+  message: 'NOT_FOUND',
```

Comment:
I would say keeping consistent with successful message. Its better to keep the error message in an error key.

And the message can be target operation. Like `User Invite`, `Campus community creation failed`. The the reason for failing can be in error key.

Now if its success error key will be null and data key will contain required data.
So something like

```ts
{
    success: bool,
    error: null | object | string | undefined,
    message: string,
    data: null | object | string | undefined
}
```